src/app.ts

```diff
@@ -0,0 +1,22 @@
+/// <reference path='../typings/tsd.d.ts' />
```

Comment:
Not sure about the file's name... It's more of a directive than an app, don't you think so ? Not that big a deal, though.